ZTors.tex

```diff
@@ -1102,6 +1102,78 @@ \subsection{Induction in $\TorZ$}\label{sec:TorZ-induction}
+After a decade of further work, this interpretation has now been extended to include a class of model categories presenting all $(\infty,1)$-topos by Shulman~\cite{shulman:univinj}, and made into an interpretation of the entire formal system by Brunerie, de Boer, Lumsdaine, and M\"{o}rtberg~\cite{initiality}.
+Thus, we can now say conclusively that the result of this paper yields a theorem about all $(\infty,1)$-toposes.
+
+However, this theorem requires a bit of unpacking to make it look familiar to higher topos theorists.
+In particular, since all $(\infty,1)$-toposes are cocomplete, the more usual way to define an internal ``circle object'' in such a topos would be as a (homotopy) colimit: specifically, the coequalizer of two copies of the identity map of the terminal object (corresponding to the presentation of a circle as a cell complex with one 0-cell and one 1-cell).
+The natural theorem to expect would then be that \emph{this} circle object is a classifier for $\ZZ$-torsors.
+
+One way to obtain this from our theorem would be to observe that according to the interpretation of higher inductive types in higher toposes constructed by~\cite{1705.07088}, the higher inductive $\Sc$ is a presentation of the above homotopy colimit.
+Since we have shown that our $\TorZ$ has the same induction principle that $\Sc$ has by definition, they must be equivalent.
+Thus, since our $\TorZ$ classifies $\ZZ$-torsors by construction, so does $\Sc$ and hence so does the circle object.
+
+However, a more direct approach is also possible, which avoids discussing higher inductive types at all:\footnote{As of this writing, higher inductive types are not yet included in the work of~\cite{initiality}.  Also the models of universes in~\cite{shulman:univinj} are not yet known to be closed under parametrized higher inductive types, although this is not a problem in our instance since $\Sc$ is not parametrized.} we can use our theorem to show that the interpretation of $\TorZ$ in an $(\infty,1)$-topos has the universal property of the homotopy colimit that defines a circle object.
+Therefore, since it classifies $\ZZ$-torsors, so does any other such homotopy colimit.
+
+Most of the work needed for this approach inside of type theory has already been done by Sojakova~\cite{sojakova:hits-hias}.
+She defines an ``$\dS$-algebra'' to be a type $C$ equipped with a point $c:C$ and a path $s:c=_C c$, and defines an $\dS$-algebra to ``have $\dS$-induction'' precisely if it satisfies the analogue of our~\eqref{eq:circle-induction} (see~\cite[Notation 22]{sojakova:hits-hias}).
+Her main theorem~\cite[Theorem 50]{sojakova:hits-hias} then implies that if $C$ has $\dS$-induction, then it is \emph{homotopy-initial} in that for any other $\dS$-algebra $(A,a,l)$ the type of ``$\dS$-homomorphisms''
+\[H_{C,A} \defeq \sum_{h:C\to A} \sum_{\beta: h(c) = a} \ap{h}(s) \ct \beta = \beta \ct l  \]
+is contractible, i.e.\ there is an element of the type
```

Comment:
Commas are needed after every "i.e.".

Reply:
Is that your convention?  I didn't see any other "i.e."s in the paper, so I went with my convention.  (Although now I see that there's one "e.g." that is followed by a comma, so I could have guessed from that.)

Reply:
I like it, because then you don't have to futz with "\ ", and line wrapping can happen.  Also, it seems to make sense in English, as well, if you think of the phrase inserted as being " ..., or, in other words, ... ".